Bottom: cf162d1fedc70e0e8d6a50b3d8d87385f9e757dc
Top:    e8ffea393ec4a8b55fcbb97983f63b77ab0ab575
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 15:04:57 +0000

IMPORTANT: DONT DRAW POUR OUTLINES DURING EXPORT

They can produce shorted boards otherwise!


---

diff --git a/src/draw.c b/src/draw.c
index f6c714b..8ad94c5 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -708,7 +708,8 @@ pour_callback (const BoxType * b, void *cl)
   struct pin_info *i = (struct pin_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  if (gui->gui)
+    DrawPour (i->Layer, pour, 0);
 
   if (pour->PolygonN)
     {
